eureka-services/src/main/java/edu/emory/cci/aiw/cvrg/eureka/services/resource/TimeUnitResource.java

```diff
@@ -48,15 +48,9 @@
 	public TimeUnitResource (TimeUnitDao inTimeUnitDao) {
 		this.timeUnitDao = inTimeUnitDao;
 	}
-
-	@GET
-	@Path("/list")
-	public List<TimeUnit> getAll () {
-		return this.timeUnitDao.getAll();
-	}
 	
 	@GET
-	@Path("/listasc")
+        @Path("")
```

Comment:
Jersey logs a warning on encountering an empty @Path. Just leave it out altogether.